fdbserver/worker.actor.cpp

```diff
@@ -1770,23 +1770,24 @@ ACTOR Future<Void> workerServer(Reference<IClusterConnectionRecord> connRecord,
 
 	folder = abspath(folder);
 
-	if (metricsPrefix.size() > 0) {
-		if (metricsConnFile.size() > 0) {
-			try {
-				state Database db =
-				    Database::createDatabase(metricsConnFile, ApiVersion::LATEST_VERSION, IsInternal::True, locality);
-				metricsLogger = runMetrics(db, KeyRef(metricsPrefix));
-				db->globalConfig->trigger(samplingFrequency, samplingProfilerUpdateFrequency);
-			} catch (Error& e) {
-				TraceEvent(SevWarnAlways, "TDMetricsBadClusterFile").error(e).detail("ConnFile", metricsConnFile);
-			}
-		} else {
-			auto lockAware = metricsPrefix.size() && metricsPrefix[0] == '\xff' ? LockAware::True : LockAware::False;
-			auto database = openDBOnServer(dbInfo, TaskPriority::DefaultEndpoint, lockAware);
-			metricsLogger = runMetrics(database, KeyRef(metricsPrefix));
-			database->globalConfig->trigger(samplingFrequency, samplingProfilerUpdateFrequency);
-		}
-	}
+	// if (metricsPrefix.size() > 0) {
+	// 	if (metricsConnFile.size() > 0) {
+	// 		try {
+	// 			state Database db =
+	// 			    Database::createDatabase(metricsConnFile, ApiVersion::LATEST_VERSION, IsInternal::True, locality);
+	// 			metricsLogger = runMetrics();
+	// 			db->globalConfig->trigger(samplingFrequency, samplingProfilerUpdateFrequency);
+	// 		} catch (Error& e) {
+	// 			TraceEvent(SevWarnAlways, "TDMetricsBadClusterFile").error(e).detail("ConnFile", metricsConnFile);
+	// 		}
+	// 	} else {
+	// 		auto lockAware = metricsPrefix.size() && metricsPrefix[0] == '\xff' ? LockAware::True : LockAware::False;
+	// 		auto database = openDBOnServer(dbInfo, TaskPriority::DefaultEndpoint, lockAware);
+	// 		metricsLogger = runMetrics();
+	// 		database->globalConfig->trigger(samplingFrequency, samplingProfilerUpdateFrequency);
+	// 	}
+	// }
+	metricsLogger = runMetrics();
```

Comment:
What is the plan with the existing metrics logging implementation? I think we should still support the old implementation of sending metric data to a separate FDB instance. There could be a parameter that controls which implementation is used.